spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :system do
+
+  let(:user) { create(:user) }
+  let(:another_user) { create(:user, email: 'another@example.com')}
+
+  describe 'ログイン後' do
+    before do
+      log_in
+      create(:task, user_id: user.id)
+    end
+    describe 'タスクの新規作成'
+      context 'フォームの入力値が正常' do
+        it 'タスクの新規作成ができること' do
+          visit new_task_path(user)
+          fill_in 'Title', with: '作成したタスク'
+          select 'todo', from: 'Status'
+          click_button 'Create Task'
+          expect(page).to have_content('Task was successfully created.')
+          expect(page).to have_content('作成したタスク')
+        end
+      context 'タイトルが未入力時に' do
+        it 'タスクの作成が失敗すること' do
+          visit new_task_path(user)
+          select 'todo', from: 'Status'
```

Comment:
Symbolで動作しないかどうか確認お願いします。
```suggestion
          select :todo, from: 'Status'
```

Reply:
0d056f0 にて修正しました。

![image](https://user-images.githubusercontent.com/52368204/80553094-96eef880-8a03-11ea-9167-f59a1db6a215.png)